contracts/Organization.sol

```diff
@@ -231,14 +233,13 @@ contract Organization is OrganizationInterface {
         onlyOwnerOrAdmin
         returns (bool success_)
     {
-        /*
-         * If the address does not change, the call is considered a success,
-         * but we don't need to emit an event as it did not actually change.
-         */
-        if (admin != _admin) {
-            emit AdminAddressChanged(_admin, admin);
-            admin = _admin;
-        }
+        require(
+            owner != _admin,
```

Comment:
this will definitely break existing code, at least in JLP

Reply:
True. It is narrowing the requirements hence affects the semantic of the function. I would mark the ticket for now as breaking change and move to "Parking Lot". Thx for heads up @schemar .